flutter/lib/src/feedback/sentry_feedback_widget.dart

```diff
@@ -0,0 +1,266 @@
+// ignore_for_file: library_private_types_in_public_api
+
+import 'package:flutter/material.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+
+class SentryFeedbackWidget extends StatefulWidget {
+  const SentryFeedbackWidget({
+    super.key,
+    this.associatedEventId,
+    this.hub,
+    this.title = 'Report a Bug',
+    this.nameLabel = 'Name',
+    this.namePlaceholder = 'Your Name',
+    this.emailLabel = 'Email',
+    this.emailPlaceholder = 'your.email@example.org',
+    this.messageLabel = 'Description',
+    this.messagePlaceholder = 'What\'s the bug? What did you expect?',
+    this.submitButtonLabel = 'Send Bug Report',
+    this.cancelButtonLabel = 'Cancel',
+    this.isRequiredLabel = '(required)',
+    this.isNameRequired = false,
+    this.isEmailRequired = false,
+  }) : assert(associatedEventId != const SentryId.empty());
+
+  final SentryId? associatedEventId;
+  final Hub? hub;
```

Comment:
If you make the widget non-const, you can initialize the hub already in the constructor:

```suggestion
 SentryFeedbackWidget({
    super.key,
    this.associatedEventId,
    Hub? hub,
    this.title = 'Report a Bug',
    this.nameLabel = 'Name',
    this.namePlaceholder = 'Your Name',
    this.emailLabel = 'Email',
    this.emailPlaceholder = 'your.email@example.org',
    this.messageLabel = 'Description',
    this.messagePlaceholder = 'What\'s the bug? What did you expect?',
    this.submitButtonLabel = 'Send Bug Report',
    this.cancelButtonLabel = 'Cancel',
    this.isRequiredLabel = '(required)',
    this.isNameRequired = false,
    this.isEmailRequired = false,
  }) : assert(associatedEventId != const SentryId.empty()), _hub = hub ?? HubAdapter();

  final SentryId? associatedEventId;
  final Hub _hub;
```

Making the constructor non-const is IMO not a big deal since this widget is probably not very frequently used, so it doesn't bring much performance improvements anyway.